universal-application-tool-0.0.1/app/views/admin/programs/ProgramBlockEditView.java

```diff
@@ -216,29 +216,39 @@ private ContainerTag blockEditPanel(
 
     ContainerTag blockInfoForm = form(csrfTag).withMethod(POST).withAction(blockUpdateAction);
 
-    blockInfoForm.with(
-        FieldWithLabel.input()
-            .setId("block-name-input")
-            .setFieldName("name")
-            .setLabelText("Block name")
-            .setValue(blockForm.getName())
-            .getContainer(),
-        FieldWithLabel.textArea()
-            .setId("block-description-textarea")
-            .setFieldName("description")
-            .setLabelText("Block description")
-            .setValue(blockForm.getDescription())
-            .getContainer(),
-        submitButton("Update Block")
-            .withId("update-block-button")
-            .withClasses(Styles.MX_4, Styles.MY_1, Styles.INLINE));
-
+    blockInfoForm
+        .withId("block-edit-form")
+        .with(
+            FieldWithLabel.input()
+                .setId("block-name-input")
+                .setFieldName("name")
+                .setLabelText("Block name")
+                .setValue(blockForm.getName())
+                .getContainer(),
+            FieldWithLabel.textArea()
+                .setId("block-description-textarea")
+                .setFieldName("description")
+                .setLabelText("Block description")
+                .setValue(blockForm.getDescription())
+                .getContainer(),
+            submitButton("Update Metadata")
+                .withId("update-block-button")
+                .withClasses(Styles.MX_4, Styles.MY_1, Styles.INLINE)
```

Comment:
Add a DISABLED type in StyleUtils and then this would be:

`.withClasses(Styles.MX_4, Styles.MY_1, Styles.INLINE, Styles.OPACITY_100, StyleUtils.disabled(Style.OPACITY_50)`

Reply:
Resolved